### DISCUSSION TITLE
I have GPU: 0 detected a defect share (potential hardware error)

### DISCUSSION THREAD

**Discussion by bilalcanaltindag** [Lolliedieb/lolMiner-releases#225] [Q&A]
I have GPU: 0 detected a defect share (potential hardware error). My none shares are posted to the pool for 3-4 days.

**Comment by DenisBoard** [comment]
> I have GPU: 0 detected a defect share (potential hardware error). My none shares are posted to the pool for 3-4 days.

I also have this problem. There is a solution?

Is it understood why?

**Comment by ChesDonno** [comment]
That is depends how you set your clock on your gpu. 'GPU: 0 detected a defect share (potential hardware error)' is due to over clocking. You may need to lower it.

tell us the name and model of your devices and your clock settings. and we'll might be able to help you from there.

**Comment by ChesDonno** [comment]
Also is it on compute mode?

**Comment by DenisBoard** [comment]
Thanks a lot, I lowered the clock speed of the GPU and its memory days ago,
and now everything is fine.

Il giorno dom 14 mar 2021 alle ore 16:46 Chesnutt ***@***.***>
ha scritto:

**Comment by Gobschibar** [comment]
I have the same problem, but i already changed the clock and memory speed to the lowest and I still get the detected defect share... any ideas? AMD Radeon r9 290...

**Comment by DenisBoard** [comment]
Good morning. If it is not solved by reducing the clocks of the gpu I do
not know how to give you an answer. I solved the problem by changing all
the hardware: main board GPU rise etc ..

Il giorno gio 14 ott 2021 alle ore 15:48 Gobschibar <
***@***.***> ha scritto:
